.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,38 @@
+## Description
+
+- Please include a summary of the change and which issue is fixed.
+
+# additional info
+- Please also include relevant  context.
+
+Closes # (issue)
+
+
+## Type of change
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+
+
```

Comment:
```suggestion

```
I think this is obvious from the description of the PR and labels added, so seems unnecessary work for the contributor